infrastructure/cdn-in-a-box/enroller/Dockerfile

```diff
@@ -20,7 +20,7 @@ FROM golang:1.14.2 AS enroller-builder
 # enroller source and dependencies
 COPY ./lib/ /go/src/github.com/apache/trafficcontrol/lib/
 COPY ./vendor/ /go/src/github.com/apache/trafficcontrol/vendor/
-COPY ./traffic_ops/client/ /go/src/github.com/apache/trafficcontrol/traffic_ops/v2-client/
+COPY ./traffic_ops/v2-client/ /go/src/github.com/apache/trafficcontrol/traffic_ops/v2-client/
```

Comment:
I'm not opposed to rebasing if another PR does it, but I run API tests using CiaB so CiaB must run in order for me to test any PR I write.

Reply:
Your situation is not unique, I'm just saying there has been at least one other person who hit this issue, and it would be better to fix the bug for everyone rather than tie it to a massive new feature. Separate bugfix PRs are also much easier to backport if necessary.